tests/test_parsers.py

```diff
@@ -2,27 +2,31 @@
 Tests for ralph.parsers module.
 """
 import gzip
-import logging
 import shutil
 
-import pandas as pd
 import pytest
 
-from ralph import filters
 from ralph.parsers import BaseParser, GELFParser
 
 
 def test_baseparser_parse():
     """Test that calling parse on the BaseParser raises a NotImplementedError."""
```

Comment:
IMO this test is useless. You can remove it.

Reply:
You are right.